src/components/statusCard/styles/StatusCard.scss

```diff
@@ -0,0 +1,54 @@
+@import '~coveo-styleguide/scss/common/palette.scss';
+@import '~coveo-styleguide/scss/common/palette-map.scss';
+@import '~coveo-styleguide/scss/variables.scss';
+
+.statusCard {
+    border-left: $status-card-border-width solid $green;
+    box-shadow: $table-shadow;
+    border-radius: $border-radius;
+    background: $pure-white;
+
+    &:global(.simple) {
+        background: transparent;
+        box-shadow: none;
+    }
+}
+
+@each $color, $value in $palette {
+    &:global(.border-color-#{$color}) {
```

Comment:
This should probably go in https://github.com/coveo/vapor/blob/b952495364c87f5e057de405c7d8f5c7666cb8eb/scss/utility/border.scss